sample_apps/1-app-scaffolding/README.md

```diff
@@ -0,0 +1,45 @@
+# App Scaffolding
+
+This sample will provides a boilerplate for standing up an Angular app that implements the ArcGIS API for JavaScript.  We leverage the Angular CLI and the [esri_loader](https://github.com/Esri/esri-loader)    
+
+## Implementing the ArcGIS API for JavaScript
+
+Using the ArcGIS API for JavaScript in an Angular applciation requires these dependencies: 
+
+* [Angular Devkit](https://github.com/angular/devkit) Development tools and libraries specialized for Angular
+* [ArcGIS API for JavaScript](https://developers.arcgis.com/javascript/)
+* [`esri-loader`](https://github.com/Esri/esri-loader) A library that helps load the [ArcGIS API for JavaScript](https://developers.arcgis.com/javascript/) into non-Dojo applications.
+* [`arcgis-js-api.d.ts`](https://github.com/Esri/jsapi-resources/tree/master/4.x/typescript) Esri TypeScript type definitions
+
+### 1. Install `esri-loader`, Esri TypeScript types, and Angular Devkit
+
+```
+  npm install
+  npm install --save esri-loader
```

Comment:
Since we've updated `package.json` to include esri-loader and arcgis types, we can remove these separate npm install steps.

Reply:
Agreed. Also let's toss out references to Angular Devkit.

 How do we want to handle individual sample app READMEs?  Each author takes on this responsibility?  We define a basic, simple one for sample app number 1, and then make all other READMEs start from there?

Reply:
@andygup can we clean up the docs once the structure is in place?  To @jwasilgeo's question, that would seem the best way to go.  Create a branch for your work in each app sample.  Make your modifications, and then provide the notes that cover the key points you're trying to demonstrate.  Cleaning up the root README in the root would seem to be the last task.  Let's wait till all the samples are in place with their respective docs.  Then we can write a nice intro doc to the entire repo that road-maps the  samples at the same time.

Reply:
>> can we clean up the docs once the structure is in place? 

Sure. My comment is also related to the dependences of `package.json`. I believe we are agreeing that our supporting libraries (esri-loader and arcgis types) are included in `package.json` and will be installed auto-majically. 

I'm okay with this...however, I also want to acknowledge there is a downside to this approach, as well. It essentially hides two important pieces from a newbie user where they might otherwise benefit from manually installing them.

Reply:
Good point as well. I’ll be sure to force a user to walk through those install steps for the esri-loader topic.